specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -3409,6 +3409,15 @@
             "read"
           ]
         },
+        "serverlessComputeCustomSubnet": {
+          "description": "The ARM resource ID for the subnet under which to create serverless compute clusters",
```

Comment:
Can you please change description to "The resource ID of an existing virtual network subnet in which serverless compute nodes should be deployed"